depends/funcs.mk

```diff
@@ -157,48 +158,60 @@ $(1)_autoconf += LDFLAGS="$$($(1)_ldflags)"
 endif
 endef
 
+shortdate="+%H:%M:%S"
+
 define int_add_cmds
-$($(1)_fetched):
+$($(1)_introduced):
+	$(AT)echo "============================================================"
+	$(AT)echo "   Package : $(1)"
+	$(AT)echo "   Date    : `date`"
+	$(AT)echo "============================================================"
+	$(AT)mkdir -p $$(@D)
+	$(AT)touch $$@
+$($(1)_fetched): | $($(1)_introduced)
 	$(AT)mkdir -p $$(@D) $(SOURCES_PATH)
 	$(AT)rm -f $$@
 	$(AT)touch $$@
-	$(AT)cd $$(@D); $(call $(1)_fetch_cmds,$(1))
+	$(AT)cd $$(@D); ( echo -n -e ""; $(call $(1)_fetch_cmds,$(1)) ) > .fetch_output 2>&1 || ( cat .fetch_output && false )
 	$(AT)cd $($(1)_source_dir); $(foreach source,$($(1)_all_sources),$(build_SHA256SUM) $(source) >> $$(@);)
 	$(AT)touch $$@
-$($(1)_extracted): | $($(1)_fetched)
-	$(AT)echo Extracting $(1)...
+$($(1)_extracted): | $($(1)_introduced) $($(1)_fetched)
```

Comment:
I should think ```$($(1)_fetched)``` would be enough here, is it not for some reason?

Reply:
The intro part would sometimes not show for packages with nothing to fetch, it appeared like. I can try again though.